src/boundingmesh/Mesh.h

```diff
@@ -101,11 +105,7 @@ class Mesh {
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
 
   // Loading/Saving files
-  void loadOff(const std::string& filename);
-  void loadObj(const std::string& filename);
-  void loadWrl(const std::string& filename, int faceset = -1,
```

Comment:
We are dropping a feature here to selectively load a faceset from a wrl, but fine.

Reply:
I can add this feature back if this is important